skbio/stats/ordination/tests/test_redundancy_analysis.py

```diff
@@ -165,6 +165,64 @@ def test_scaling2(self):
+    # with vegan's (module multiplying by a constant). I can also
+    # compute scaling 2, agreeing with vegan, but there are no written
+    # results in L&L.
+    def setUp(self):
+        """varespec and varechem from Väre etal. 1995 DOI: 10.2307/3236351"""
+
+        self.Y = read_csv(get_data_path('varespec.csv'))
+        self.X = read_csv(get_data_path('varechem.csv'))
+
+    def test_biplot_score(self):
+
+        rda_ = rda(y=varespec, x=varechem, scale_Y=False, scaling=1)
+
+        # Load data as computed with vegan 2.4-3:
+        # library(vegan)
+        # data(varechem)
+        # data(varespec)
+        # rda_ = rda(X=varespec, Y=varechem, scale=FALSE)
+        # write.table(summary(rda_, scaling=1)$biplot, 'vare_rda_biplot_from_vegan.csv', sep=',')
+        # write.table(summary(rda_, scaling=1)$sites, 'vare_rda_sites_from_vegan.csv', sep=',')
+        # write.table(summary(rda_, scaling=1)$species, 'vare_rda_species_from_vegan.csv', sep=',')
+        # write.table(summary(rda_, scaling=1)$constraints, 'vare_rda_constraints_from_vegan.csv', sep=',')
+        # write.table(summary(rda_, scaling=1)$cont$importance[2, ], 'vare_rda_propexpl_from_vegan.csv', sep=',')
+        # write.table(summary(rda_, scaling=1)$cont$importance[1, ], 'vare_rda_eigvals_from_vegan.csv', sep=',')
+
+        vegan_features = pd.read_csv(get_data_path('vare_rda_species_from_vegan.csv'))
+        vegan_samples = pd.read_csv(get_data_path('vare_rda_sites_from_vegan.csv'))
+        vegan_biplot = pd.read_csv(get_data_path('vare_rda_biplot_from_vegan.csv'))
+        vegan_constraints = pd.read_csv(get_data_path('vare_rda_constraints_from_vegan.csv'))
+        vegan_propexpl = pd.read_csv(get_data_path('vare_rda_propexpl_from_vegan.csv'))
+        vegan_propexpl = pd.Series(vegan_propexpl.x.values, index=rda_.eigvals.index)
+        vegan_eigvals = pd.read_csv(get_data_path('vare_rda_eigvals_from_vegan.csv'))
+        vegan_eigvals = pd.Series(vegan_eigvals.x.values, index=rda_.eigvals.index)
+
+        # transform the output of rda_ to match column selection of vegan
+        rda_ = OrdinationResults(
+            'RDA', 'Redundancy Analysis',
+            samples=rda_.samples.iloc[:,0:6],
+            features=rda_.features.iloc[:,0:6],
+            sample_constraints=rda_.sample_constraints.iloc[:,0:6],
+            biplot_scores=rda_.biplot_scores.iloc[:,0:6],
+            proportion_explained=rda_.proportion_explained,
+            eigvals=rda_.eigvals)
+
+        exp = OrdinationResults(
+            'RDA', 'Redundancy Analysis',
+            samples=vegan_samples,
+            features=vegan_features,
+            sample_constraints=vegan_constraints,
+            biplot_scores=vegan_biplot,
+            proportion_explained=vegan_propexpl,
+            eigvals=vegan_eigvals)
+
+        assert_ordination_results_equal(rda_, exp,
+                                        ignore_directionality=True,
+                                        decimal=6)
```

Comment:
If the above two comments are corrected, the results here don't seem to agree ...

Reply:
biplot_scores are still different from vegan (only the first score column is equal). Any help to figure out why would be appreciated.